src/main/java/org/apache/commons/lang3/builder/EqualsBuilder.java

```diff
@@ -34,7 +34,7 @@
  *
  * <p> This class provides methods to build a good equals method for any
  * class. It follows rules laid out in
- * <a href="http://www.oracle.com/technetwork/java/effectivejava-136174.html">Effective Java</a>
+ * <a href="https://www.oracle.com/technetwork/java/effectivejava-136174.html">Effective Java</a>
```

Comment:
Redirects to https://www.oracle.com/java/technologies/effectivejava.html

Reply:
@marcwrobel ping?

Reply:
@garydgregory, I cannot update the PR for the time being. Feel free to apply the necessary changes.